core/plugins/org.fusesource.ide.foundation.ui/src/org/fusesource/ide/foundation/ui/views/TabFolderSupport2.java

```diff
@@ -162,6 +162,10 @@ public IContributor getContributor() throws InvalidRegistryObjectException {
 							return null;
 						}
 
+						public int getHandleId() {
```

Comment:
Please remove this method. It will be added by https://github.com/jbosstools/jbosstools-fuse/pull/1355

Reply:
Yes, please remove